README.md

```diff
@@ -1398,6 +1400,53 @@ You can use these placeholders to create custom queries to filter and migrate a
 
 > When creating custom queries via this job, please remember to confirm that B2C Customer Profiles exist that satisfy your constructed query.  Any query that returns zero results will generate a NO DATA message in the job log-files.
 
+### Data mapping between B2C Commerce and the Core Platform
+
+As the B2C Commerce customer profile and its addresses are fetched by the core platform, the core platform holds the data-mapping between the core attributes and the B2C ones. You can find hereafter the detail of this mapping. The `Core Alt ID` is used in place of the `Core ID` when person accounts is enabled. The `Core ID` is used when person accounts are not enabled:
+
+| Label               | Core Object            | Core ID                    | Core Alt ID                 | B2C Object      | OCAPI ID            |
+|:-------------------:|:----------------------:|:--------------------------:|:---------------------------:|:---------------:|:-------------------:|
+| Address Id          | B2C_CustomerAddress__x | AddressId__c               | (None)                      | CustomerAddress | address_id          |
```

Comment:
This is great. Does the __x here represent AddressId? So B2C_CustomerAddress__AddressId?

Reply:
No. Objects ended with `__x` are [External objects](https://developer.salesforce.com/docs/atlas.en-us.234.0.object_reference.meta/object_reference/sforce_api_objects_external_objects.htm). These are used with the Salesforce Connect feature.
In order to share the data mapping for addresses along with both technical solutions implemented within b2c-crm-sync:
1. Sync through Salesforce Connect
2. Fetch through flow and custom LWC
We reused the same "B2C_CustomerAddress__x" core object within the mapping configuration.

In fact, when the Salesforce Connect feature is used, a real `B2C_CustomerAddress__x ` object is created, and addresses from B2C Commerce are queried in realtime and store in Core. Real records are created here in the Core Platform.
As this Salesforce Connect feature is a paid license, and we don't want users being constrained by any license in order to use b2c-crm-sync, we also provided a flow+custom LWC way of syncing addresses.
This way is fetching addresses in real-time when seeing a contact/personAccount record in core, but no record is being created in the core platform.

Hope this helps